services/movie_session.py

```diff
@@ -3,6 +3,14 @@
 from db.models import MovieSession
 
 
+def get_taken_seats(movie_session_id: int) -> list:
+    return list(
+        MovieSession.objects.get(id=movie_session_id).tickets.values(
+            "row", "seat"
+        )
+    )
+
+
 def create_movie_session(
     movie_show_time: str, movie_id: int, cinema_hall_id: int
```

Comment:
Consider using `datetime` objects for `movie_show_time` instead of strings to ensure proper handling of date and time data. This will also help avoid potential issues with date formatting and time zone handling.